csi/controller_server.go

```diff
@@ -456,14 +463,17 @@ func generateAttachmentID(volName, nodeID string) string {
 
 // publishVolume sends the actual attach request to the longhorn api and executes the passed waitForResult func
 func (cs *ControllerServer) publishVolume(volume *longhornclient.Volume, nodeID, attachmentID string, waitForResult func() error) (*csi.ControllerPublishVolumeResponse, error) {
+	log := getLoggerForCSIControllerServer()
+	log = log.WithFields(logrus.Fields{"function": "DeleteVolume"})
```

Comment:
This is incorrect. Please review all changes.

Reply:
Fixed and checked, thanks!